Makefile

```diff
@@ -21,7 +21,12 @@ PACKAGE_DIRECTORIES := $(PACKAGE_LIST) | sed 's|github.com/ti-community-infra/$(
 FILES     := $$(find $$($(PACKAGE_DIRECTORIES)) -name "*.go")
 
 
-.PHONY: clean test dev check tidy
+.PHONY: clean test cover fmt tidy staticcheck dev check
```

Comment:
Why should `fmt` and `tidy` be performed after `test`?

Reply:
It doesn't matter, the order doesn't affect the execution.

Reply:
I try to mock it, the execution order is: 
![image](https://user-images.githubusercontent.com/5086433/109099450-1a7d9100-775e-11eb-95f9-1310b1c14941.png)

Reply:
I think we shouldn't make .PHONY.

Please refer to https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html. PHONY is a fake target.

Reply:
OK, Thank you for your explanation!